gtests/net/mptcp/fastclose/receive_fastclose_with_rst_multi_v4.pkt

```diff
@@ -16,10 +16,9 @@
 
 // add_addr
 +0.0    >                                .  1:1(0)        ack 1                <add_address addr[saddr1] hmac=auto, dss dack4=1                    dll=0    nocs>
-// TODO: send echo bit
-// +0.0 <                                .  1:1(0)        ack 1     win 256    <add_address addr[saddr1],           dss dack8=1                    dll=0    nocs>
++0.0    <                                .  1:1(0)        ack 1     win 256    <add_address addr[saddr1] addr_echo, dss dack4=1                    dll=0    nocs>
 
-+0.2    <                               P.  1:1001(1000)  ack 1     win 450    <nop, nop,                           dss dack8=1    dsn8=1    ssn=1 dll=1000 nocs>
++0.2    <                               P.  1:1001(1000)  ack 1     win 450    <nop, nop,                           dss dack8=4    dsn8=1    ssn=1 dll=1000 nocs>
```

Comment:
why are we here at `dack8=4`?

Reply:
we aren't. That was meant to be a `dack4=1` but i wrote the `4` at a wrong offset.